lucene/core/src/java/org/apache/lucene/index/DefaultIndexingChain.java

```diff
@@ -52,7 +52,7 @@
 /** Default general purpose indexing chain, which handles
  *  indexing all types of fields. */
 final class DefaultIndexingChain extends DocConsumer {
-  final Counter bytesUsed;
+  final Counter bytesUsed = Counter.newCounter();
```

Comment:
So we are breaking out a separate `Counter` for `DefaultIndexingChain`, from `DWPT`, when previously they shared the same counter?

Reply:
exactly that's the main change here otherwise we need to do back and forth accounting inside the StoredFieldsWriter